tests/unit/UnreadIndicatorUpdaterTest.ts

```diff
@@ -0,0 +1,41 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+import CONST from '../../src/CONST';
+import getUnreadReportsForUnreadIndicator from '../../src/libs/UnreadIndicatorUpdater';
+
+describe('UnreadIndicatorUpdaterTest', () => {
+    describe('should return correct number of unread reports', () => {
+        it('given last read time < last visible action created', () => {
+            const reportsToBeUsed = {
+                1: {reportID: '1', reportName: 'test', type: CONST.REPORT.TYPE.EXPENSE, lastReadTime: '2023-07-08 07:15:44.030', lastVisibleActionCreated: '2023-08-08 07:15:44.030'},
+                2: {reportID: '2', reportName: 'test', type: CONST.REPORT.TYPE.TASK, lastReadTime: '2023-02-05 09:12:05.000', lastVisibleActionCreated: '2023-02-06 07:15:44.030'},
+                3: {reportID: '3', reportName: 'test', type: CONST.REPORT.TYPE.TASK},
+            };
+            expect(getUnreadReportsForUnreadIndicator(reportsToBeUsed, '4').length).toBe(2);
```

Comment:
`getUnreadReportsForUnreadIndicator` depends on `currentReportId`, right? In our tests, it always points to a different, outside report (id: 4). Isn't this a hole?

Reply:
I have changed it so that its no longer pointing to an outside report.